<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" columnstretch="2,1,0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="8" column="1">
      <widget class="QPushButton" name="highScoreBtn">
       <property name="text">
        <string>View High Scores</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="buyHealthBtn">
       <property name="text">
        <string>Buy Heath</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QPushButton" name="saveSelectedShipsBtn">
       <property name="text">
        <string>Save selected ships</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2" colspan="2">
      <widget class="QLabel" name="versionLabel">
       <property name="text">
        <string>v1.0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="2" colspan="2">
      <widget class="QLabel" name="authorLabel">
       <property name="text">
        <string>Author: Hai &amp; Sang</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="11">
      <widget class="QGraphicsView" name="galaxyView"/>
     </item>
     <item row="0" column="1" rowspan="6">
      <widget class="QListView" name="shipInStarSystemListView"/>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="endTurnBtn">
       <property name="text">
        <string>End Turn</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2" rowspan="2" colspan="2">
      <widget class="QLabel" name="gameNameLabel">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;GAME NAME&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="healthLabel">
       <property name="text">
        <string>Health</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLCDNumber" name="healthLCDNumber"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="creditLabel">
       <property name="text">
        <string>Credits</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLCDNumber" name="creditLCDNumber"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="pointLabel">
       <property name="text">
        <string>Points</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLCDNumber" name="pointLCDNumber"/>
     </item>
     <item row="10" column="2" colspan="2">
      <widget class="QPushButton" name="viewCreditsBtn">
       <property name="text">
        <string>Credits</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>762</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGAME_NAME">
    <property name="title">
     <string>GAME NAME</string>
    </property>
   </widget>
   <addaction name="menuGAME_NAME"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
